MPXPLAY/AU_CARDS/SC_ICH.C

```diff
@@ -322,7 +351,6 @@ static void snd_intel_prepare_playback(struct intel_card_s *card,struct mpxplay_
  snd_intel_write_32(card,ICH_PO_BDBAR_REG,(uint32_t)pds_cardmem_physicalptr(card->dm,table_base));
 
  snd_intel_write_8(card,ICH_PO_LVI_REG,(ICH_DMABUF_PERIODS-1)); // set last index
- snd_intel_write_8(card,ICH_PO_CIV_REG,0); // reset current index
```

Comment:
The CIV register is read-only in AC97, so I removed writes to it.

Reply:
Yeah I saw this too, but I didn't remove it, thought it wouldn't hurt. And yes remove it also wouldn't hurt.